drivers/gpu/drm/panel/panel-sharp-y030xx067a.c

```diff
@@ -210,16 +301,16 @@ static int y030xx067a_remove(struct spi_device *spi)
 
 static const struct drm_display_mode y030xx067a_modes[] = {
 	{
-		.clock = 54000,
+		.clock = 43200,
 		.hdisplay = 320,
-		.hsync_start = 320 + 128,
-		.hsync_end = 320 + 128 + 28,
-		.htotal = 320 + 128 + 28 + 25,
+		.hsync_start = 320 + 100,
+		.hsync_end = 320 + 100 + 30,
+		.htotal = 320 + 100 + 30 + 35,
 		.vdisplay = 480,
-		.vsync_start = 480 + 36,
-		.vsync_end = 480 + 36 + 1,
-		.vtotal = 480 + 36 + 1 + 16,
-		.flags = DRM_MODE_FLAG_PHSYNC | DRM_MODE_FLAG_PVSYNC,
+		.vsync_start = 480 + 124,
+		.vsync_end = 480 + 124 + 20,
+		.vtotal = 480 + 124 + 20 + 16,
+		.flags = DRM_MODE_FLAG_NHSYNC | DRM_MODE_FLAG_NVSYNC,
```

Comment:
This is actually not correct. From these timings, the DRM core will compute that you have a `43,2 MHz / ((320+100+30+35)*(480+124+20+16)) == 139,18 Hz` panel. The fact that you do get a 60.0 Hz image from these timings is because the ingenic-drm driver doesn't compute the right hardware values in the case where a serial 8bpp panel is used. Hence the "x3" added there (but this might be wrong).

Reply:
Here are timings/hposition that work keeping the "x3"s in the ingenic drm driver (keeping the reg_map(priv->map, 0x90, 0x33) ; 

{ 0x04, REG04_HPOSITION1(0xd1) }

static const struct drm_display_mode y030xx067a_modes[] = {
	{
		.clock = 54000,
		.hdisplay = 320,
		.hsync_start = 320 + 90,
		.hsync_end = 320 + 90 + 37,
		.htotal = 320 + 90 + 37 + 33,
		.vdisplay = 480,
		.vsync_start = 480 + 109,
		.vsync_end = 480 + 109 + 20,
		.vtotal = 480 + 109 + 20 + 16,
		.flags = DRM_MODE_FLAG_NHSYNC | DRM_MODE_FLAG_NVSYNC,
	},
};

480x625x60x3=54M